proto.com.unity.formats.alembic/Runtime/Scripts/Misc/Bool.cs

```diff
@@ -3,10 +3,12 @@
 
 namespace UnityEngine.Formats.Alembic.Sdk
 {
-    // bool is marshal as int (4 byte) by default and you need ugly [MarshalAs(UnmanagedType.U1)] to pass to (or receive from) C++ code.
-    // this struct emulates bool and marshal as byte (1 byte). this makes things bit easier in some cases.
+    /// <summary>
+    /// bool is marshal as int (4 byte) by default and you need ugly [MarshalAs(UnmanagedType.U1)] to pass to (or receive from) C++ code.
```

Comment:
`Alembic marshals bool as 1 byte, whereas the C# default is to marshal bool as 4 bytes. This struct implements 1-byte marshalling.`